### PR TITLE
transmit log rotation max size to nydus daemon

### PR DESCRIPTION
**Pull request by sctb512**
Currently, the maximum log rotation size is not passed to the `nydus` daemon. It would be better to allow users to set this value.

### CODE REVIEW

File: pkg/daemon/daemon.go [hunk 1]
```diff
@@ -39,14 +39,15 @@ type NewDaemonOpt func(d *Daemon) error
...
-	LogLevel    string
-	LogToStdout bool
-	DaemonMode  config.DaemonMode
-	FsDriver    string
+	ID              string
+	ProcessID       int
+	APISocket       string
+	LogDir          string
+	LogLevel        string
+	LogRotationSize int
```
[the remark this answers is not in the document]
Comment (sctb512) [reply pkg/daemon/daemon.go:47]: I think this entry should follow the store way of `LogLevel`.

[the remark this answers is not in the document]
Comment (changweige) [reply pkg/daemon/daemon.go:47]: It's fine. By adding `LogRotationSize` to the DB record, we have to consider compatibility. It's possible to be zero when loading from an old nydusd record, and how do you make it apply to an old running nydusd if you never update its DB record. Moverover, if it's zero we don't have to append the argument `log-rotation-size` to CLI

[the remark this answers is not in the document]
Comment (sctb512) [reply pkg/daemon/daemon.go:47]: > if it's zero we don't have to append the argument `log-rotation-size` to CLI

Yes, you are right. Fixed. Now, the `LogRotationSize` will be omitted if it is zero when build command arguments.

**Review by changweige — APPROVED** [review]
LGTM

**Pull request merged**